tools/azure-sdk-tools/devtools_testutils/azure_testcase.py

```diff
@@ -101,6 +102,7 @@ def __init__(self, method_name, config_file=None,
         config_file = config_file or os.path.join(self.working_folder, TEST_SETTING_FILENAME)
         if not os.path.exists(config_file):
             config_file = None
+        load_dotenv(find_dotenv())
```

Comment:
What type of env variables do we expect to be loaded? We need to be very careful about having secrets in env files that might get committed to the repo.

Reply:
I will add the .env file to the .gitignore as well. We're expecting devs to put any variables that they use for authenticating so it will be important that the `.env` file is not committed to the repo.

Reply:
Adding it to the .gitignore is a good helper step but we should try and encourage folks to not even put the .env file under the git repo root. They should go into the user protected settings somewhere.

Reply:
I agree that is a great way to do it, I ran a quick test and placing my `.env` file in one directory before `azure-sdk-for-python` the environment variables loaded perfectly.

Reply:
Great so when we document this we should suggest people put that file outside of the repo. Preferably in a user directory if that is a probing path of the python-dotenv library but if not at least in a higher level would be more secure.